up42/auth.py

```diff
@@ -111,10 +111,10 @@ def _endpoint(self) -> str:
     def _get_token(self):
         try:
             self._get_token_project()
-        except requests.exceptions.HTTPError:
+        except requests.exceptions.HTTPError as err:
             raise ValueError(
```

Comment:
What does this `from err` do? This is a new notation for me :)

Reply:
Similar to the stacktrace in the screenshot example, is a bit more explicit that this custom Exception is a direct consequence of the previous exception. Is a detail but fits well here I think.